app/src/androidTest/java/ch/epfl/sweng/swenggolf/WaitingActivityTest.java

```diff
@@ -0,0 +1,94 @@
+package ch.epfl.sweng.swenggolf;
+
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import ch.epfl.sweng.swenggolf.database.Database;
+
+import ch.epfl.sweng.swenggolf.database.DbError;
+import ch.epfl.sweng.swenggolf.database.DatabaseUser;
+import ch.epfl.sweng.swenggolf.database.FakeDatabase;
+import ch.epfl.sweng.swenggolf.database.ValueListener;
+
+import ch.epfl.sweng.swenggolf.main.MainActivity;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+@RunWith(AndroidJUnit4.class)
+public class WaitingActivityTest {
+
+    private static final String name = "Hello";
+    private static final String mail = "Hello@World.ok";
+    private static final String uid1 = "1234";
+    private static final String uid2 = "5678";
+
+    private static final String photo = "photo";
+
+
+    private static final User userdb = new User(name, uid1 , mail, photo);
+    private static final User usernotdb = new User(name, uid2,mail, photo);
```

Comment:
All these variables are `static final`. They would follow more closely the style rules if they were written in uppercase.